src/main.js

```diff
@@ -0,0 +1,42 @@
+function strComp (str1, str2) {
+  if (str1.localeCompare(str2) === -1 || str1.localeCompare(str2) === 0){
```

Comment:
We can simplify this to:

```javascript
return str1.localeCompare(str2) <= 0
```

---

Based on the [documentation of `localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare), its return value is:

> A negative number if the reference string occurs before the compare string; positive if the reference string occurs after the compare string; 0 if they are equivalent.